.github/workflows/ci.yml

```diff
@@ -3,16 +3,16 @@ name: Ubuntu CI
 on: [push, pull_request]
```

Comment:
@Athanaseus I forgot to ask, can you add a weekly cronjob here.

It's probably possible to just copy this from here

https://github.com/ratt-ru/xarray-ms/blob/fa8ba63f0dde23686b9e786473f76f8843b55688/.github/workflows/ci.yml#L10-L11

Reply:
Thank you @sjperkins, cronjon added.